### ISSUE TITLE
the accuracy about pretrained models?

### ISSUE THREAD

**Issue by lunalulu** [wangziren1/pytorch_pose_proposal_network#7]
i want to know the accuracy of the pretrained models resnet18 and resnet50?
thanks
